servers/src/test/java/tachyon/master/next/filesystem/meta/InodeDirectoryTests.java

```diff
@@ -0,0 +1,188 @@
+  }
+
+  @Test
+  public void batchRemoveChildTest() {
+    InodeDirectory inodeDirectory = createInodeDirectory();
+    InodeFile inodeFile1 = createInodeFile(1);
+    InodeFile inodeFile2 = createInodeFile(2);
+    InodeFile inodeFile3 = createInodeFile(3);
+    inodeDirectory.addChild(inodeFile1);
+    inodeDirectory.addChild(inodeFile2);
+    inodeDirectory.addChild(inodeFile3);
+    Assert.assertEquals(3, inodeDirectory.getNumberOfChildren());
+    inodeDirectory.removeChild("testFile1");
+    Assert.assertEquals(2, inodeDirectory.getNumberOfChildren());
+    Assert.assertFalse(inodeDirectory.getChildrenIds().contains(createBlockId(1)));
+  }
+
+  @Test
+  public void equalsTest() {
+    InodeDirectory inode1 = new InodeDirectory("test1", 1, 0, System.currentTimeMillis());
+    InodeDirectory inode2 = new InodeDirectory("test2", 1, 0, System.currentTimeMillis());
+    InodeDirectory inode3 = new InodeDirectory("test3", 3, 0, System.currentTimeMillis());
+    Assert.assertTrue(inode1.equals(inode2));
+    Assert.assertTrue(inode1.equals(inode1));
+    Assert.assertFalse(inode1.equals(inode3));
+  }
+
+  @Test
+  public void getIdTest() {
+    Assert.assertEquals(1, createInodeDirectory().getId());
+  }
+
+  @Test
+  public void isDirectoryTest() {
+    Assert.assertTrue(createInodeDirectory().isDirectory());
+  }
+
+  @Test
+  public void isFileTest() {
+    Assert.assertFalse(createInodeDirectory().isFile());
+  }
+
+  @Test
+  public void removeChildTest() {
+    InodeDirectory inodeDirectory = createInodeDirectory();
+    InodeFile inodeFile1 = createInodeFile(1);
+    inodeDirectory.addChild(inodeFile1);
+    Assert.assertEquals(1, inodeDirectory.getNumberOfChildren());
+    inodeDirectory.removeChild(inodeFile1);
+    Assert.assertEquals(0, inodeDirectory.getNumberOfChildren());
+  }
+
+  @Test
+  public void removeNonExistentChildTest() {
+    InodeDirectory inodeDirectory = createInodeDirectory();
+    InodeFile inodeFile1 = createInodeFile(2);
+    InodeFile inodeFile2 = createInodeFile(3);
+    inodeDirectory.addChild(inodeFile1);
+    Assert.assertEquals(1, inodeDirectory.getNumberOfChildren());
+    inodeDirectory.removeChild(inodeFile2);
+    Assert.assertEquals(1, inodeDirectory.getNumberOfChildren());
+  }
+
+  @Test
+  public void reverseIdTest() {
+    InodeDirectory inode1 = createInodeDirectory();
+    inode1.reverseId();
+    Assert.assertEquals(-1, inode1.getId());
+  }
+
+  @Test
+  public void sameIdChildrenTest() {
+    InodeDirectory inodeDirectory = createInodeDirectory();
+    InodeFile inodeFile1 = createInodeFile(1);
+    inodeDirectory.addChild(inodeFile1);
+    inodeDirectory.addChild(inodeFile1);
+    Assert.assertTrue(inodeDirectory.getChildrenIds().get(0) == createBlockId(1));
+    Assert.assertEquals(1, inodeDirectory.getNumberOfChildren());
+  }
+
+  @Test
+  public void setLastModificationTimeTest() {
+    long createTimeMs = System.currentTimeMillis();
+    long modificationTimeMs = createTimeMs + 1000;
```

Comment:
I think you can use `Constants.SECOND_MS` instead of 1000 when you want to refer to a second in milliseconds (applies in other parts of the code as well)